buildSrc/src/main/java/Versions.kt

```diff
@@ -0,0 +1,86 @@
+object Versions {
+  const val minSdk = 21
+  const val compileSdk = 30
+  const val kotlin = "1.4.32"
```

Comment:
I'm curious, what version of Kotlin does Gradle use for the build script itself?

Reply:
It uses an embedded Kotlin and the version depends on the Gradle plugin version. In this scenario, it's `1.4.31`
https://docs.gradle.org/current/userguide/kotlin_dsl.html#sec:kotlin